src/utilities/permissions.ts

```diff
@@ -332,6 +332,9 @@ const queryPermissions: Record<GQLKeys, (user: User | null, ...args: any[]) => b
   CREATE_CONSTRAINT_MODEL_SPECIFICATION: (user: User | null): boolean => {
     return isUserAdmin(user) || getPermission([Queries.INSERT_CONSTRAINT_MODEL_SPECIFICATION], user);
   },
+  CREATE_DERIVATION_GROUP: (user: User | null): boolean => {
```

Comment:
For most of these new query permissions except for external event related ones, they don't seem to have a corresponding `featurePermission` that is used within the UI to disable form elements and provide permission denial messages when they don't have permission. The current convention on the UI is to prevent users from initiating actions that would result in permission errors from the backend as much as possible. A good example to follow is [ActivityDirectiveForm.svelte](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/src/components/activity/ActivityDirectiveForm.svelte). The two ways to integrate a permission into elements or components are the following:
1. [For basic html elements](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/src/components/activity/ActivityDirectiveForm.svelte#L516)
2. [For components that have the "use" prop](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/src/components/activity/ActivityDirectiveForm.svelte#L574)

Reply:
Believe I got all the new queries properly permission'd including w. ownership for `ExternalSource` and `DerivationGroup`, let me know if you see anything missing or off!